integration_tests/__tests__/__snapshots__/failures.test.js.snap

```diff
@@ -70,6 +70,59 @@ exports[`not throwing Error objects 4`] = `
 "
 `;
 
+exports[`works with assertions in separate files 1`] = `
+"FAIL __tests__/test_macro.test.js
+  ✕ use some imported macro to make assertion
+
+  ● use some imported macro to make assertion
+
+    expect(received).toEqual(expected)
+    
+    Expected value to equal:
+      2
+    Received:
+      1
+
+      10 | 
+      11 | module.exports = (one: any, two: any) => {
+    > 12 |   expect(one).toEqual(two);
```

Comment:
Potentially we can walk down the stack until we find the test file. But I think this makes more sense